.gitignore

```diff
@@ -1,2 +1,4 @@
 /target
+*/target
 errors.err
+Cargo.lock
```

Comment:
It's useful to check Cargo.lock in to have reliable CI and stuff.

Reply:
Ah, that was by mistake. Thanks.